### PR TITLE
fix: update dependencies

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,42 +51,42 @@
...
-		"@typescript-eslint/eslint-plugin": "^5.58.0",
-		"@typescript-eslint/parser": "^5.58.0",
-		"del-cli": "^5.0.0",
-		"eslint": "^8.38.0",
-		"eslint-config-prettier": "^8.8.0",
-		"eslint-doc-generator": "^1.4.3",
-		"eslint-import-resolver-typescript": "^3.6.1",
-		"eslint-plugin-import": "^2.29.0",
-		"eslint-plugin-jest": "^27.2.1",
+		"@types/node": "^16.18.113",
```
Comment (Belco90) [inline package.json:65]: This one should actually be v20 since we changed the version in `nvmrc`. Could you change that here?

File: package.json [hunk 2]
```diff
@@ -51,42 +51,42 @@
...
+		"eslint-remote-tester": "^3.0.1",
 		"eslint-remote-tester-repositories": "^1.0.1",
 		"husky": "^8.0.3",
 		"is-ci": "^3.0.1",
 		"jest": "^28.1.3",
-		"lint-staged": "^13.2.1",
-		"npm-run-all": "^4.1.5",
-		"prettier": "2.8.7",
+		"lint-staged": "^13.3.0",
+		"npm-run-all2": "^5.0.2",
```
Comment (Belco90) [inline package.json:85]: What is this `npm-run-all2`?

**Review by Belco90 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (MichaelDeBoey) [reply package.json:85]: It's the maintained version of `npm-run-all`

Comment (Belco90) [reply package.json:85]: Oh, interesting. Hopefully, after releasing the new major and upgrading all core dependencies, I'll be able to migrate to pnpm, which provides utils to run several scripts in a similar way.

**Comment by Belco90** [comment]
We are having issues with Node v12. Perhaps is better just to wait for the next plugin major, where we drop support for outdated Node versions.

**Comment by MichaelDeBoey** [comment]
@Belco90 Node 12 seems to be fixed now

**Review by Belco90 — APPROVED** [review]

**Review by Belco90 — APPROVED** [review]

**Pull request merged**